tests/examples/runner.sh

```diff
@@ -38,6 +38,12 @@ then
     exit 1
 fi
 
+# Automatically add the output for new examples
+if [ ! -e "${expected_output}" ]
+then
+    cp "${file_stdout}" "${expected_output}"
```

Comment:
How about `echo "${expected_output} did not yet exist and was created"; exit 1`? It's not much since the next run will just succeed, but it might help in making it more noticable that this file was created?
But the current version is fine with me as well. We can bikeshed this later. ;-)

Reply:
Ok, I will merge as-is given the now 3 proposed changes are probably better suited after after #2333 is merged. Doing so right now in the configure phase isn't the same as doing it during `make`. Thus this commit fixes an immediate issue ("adding new tests is a PITA") while #2333 fixes "adding more tests is slowing down the build and is a PITA".